src/envoy/http/backend_auth/filter.cc

```diff
@@ -43,11 +43,11 @@ RegisterCustomInlineHeader<CustomInlineHeaderRegistry::Type::RequestHeaders>
     authorization_handle(CustomHeaders::get().Authorization);
 
 struct RcDetailsValues {
-  // The request is rejected due to missing backend auth token in internal
-  // filter config.
-  const std::string MissingBackendToken = "missing_backend_token";
-  // The request is rejected due to missing operation in internal filter state.
-  const std::string MissingOperation = "missing_operation";
+  // Missing backend auth token in internal filter config.
+  const std::string MissingBackendToken = "backend_auth_missing_backend_token";
```

Comment:
Can we have some common utility function shared across the filters to form the RC detail?

Some function that does `absl::StrCat(filter_name, error_type, "{", error_name, "}").

I'm specifically worried about `error_type`, it should be shared across filters. `bad_request`, `missing_operation` are all common.

Reply:
Good point, done.